packages/discovery-react-components/src/components/DocumentPreview/components/PdfViewerWithHighlight/PdfViewerWithHighlight.tsx

```diff
@@ -99,32 +101,51 @@ const PdfViewerWithHighlight = forwardRef<any, Props>(
 
     const setCurrentErrMsgFromPdfConst = useIsPfdError(_isPdfRenderError, setIsPdfRenderError);
 
+    // TODO: Dynamically create a style for every category. Match color of category
```

Comment:
Does this still need to be a TODO? Looks like you implemented this, so you could just make it a descriptive comment